scripts/listapranzo.js

```diff
@@ -85,6 +85,29 @@ module.exports = function (robot) {
     order.users[user.id] = dishes;
   };
 
+  var fuzzyMatch = function (dish, menuline) {
+    dish = dish.trim().toLowerCase().replace(" ", "");
+    var key = "";
+    for (var i = 0; i < dish.length; i++)
+      key = key + dish[i] + ".*"
```

Comment:
Si come substring dovresti fare split su spazi dell'input dell'utente, e poi fare substring in maniera ordinata delle varie sottoparole (non matchare items che le hanno tutte ma in ordine diverso). Secondo me fuzzy è più comodo in ogni caso, ma tant'è.